### DISCUSSION TITLE
Can't focus/click other elements if the dropdown menu from List in open

### DISCUSSION THREAD

**Discussion by alexnatea** [tailwindlabs/headlessui#3283] [Help] [answered]
Version: @Headlessui 2.0.4 

I want to focus/click the other elements on page when the dropdown is open instead of clicking twice (first click is to exit dropdown and the other click on the other elements)

https://github.com/tailwindlabs/headlessui/assets/76999862/704210f0-d7cc-4f56-98ae-f326e5962a62

Here is the code:

![image](https://github.com/tailwindlabs/headlessui/assets/76999862/3a160f10-ae5a-4bb3-bbb8-6ecadd735053)

**Comment by reghope** [comment]
Having the same issue

**Comment by reghope** [comment, marked as answer]
@alexnatea Try modal={false} as a Headless.Menu prop, or you can try my workaround for making it close in 1 click: https://github.com/tailwindlabs/headlessui/discussions/3299

**Reply by bernsteinmatt** [reply]
Setting modal={false} worked! Thanks!

**Reply by alexnatea** [reply]
Indeed. setting modal={false} to my ListboxOptions element solved my issue. Thank you!!!
